homes_guests_loves.html

```diff
@@ -0,0 +1,121 @@
+        <p class="home-destination home-text">Santa Cruz de Tenerife, Spain</p>
+        </div>
+            <div class="home-guests-box box">
+        <img class="home-guests-photo photo" src="images/homes_images/villa_kunerad.svg" alt="villa_kunerad">
+        <p class="home-hotel-name home-text">Villa Kunerad</p>
+        <p class="home-destination home-text">Vysokie Tatry, Slowakia <span class="hidden">destination</span></p>
+            </div>
+                <div class="home-guests-box box">
+        <img class="home-guests-photo photo" src="images/homes_images/hostel_friendship.svg" alt="hostel_friendship">
+        <p class="home-hotel-name home-text">Hostel Friendship</p>
+        <p class="home-destination home-text">Berlin, Germany <span class="hidden">destination</span></p>
+                </div>
+        <div class="home-switcher">
+            <img class="home-switcher-left" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+            <img class="home-switcher-right" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+        </div>
+
+    </div>
+    </div>
+
+</section>
+
+<section class="best-destinations">
+    <div class="best-destinations-container">
+    <h1 class="col-lg-12 col-sm-8">The best destinations</h1>
+    <form>
+            <div class="best-destinations-btns">
+        <button class="best-destinations-btn-regions btn">Regions</button>
+        <button class="best-destinations-btn-cities btn">Cities</button>
+                <button class="best-destinations-btn-places btn">Places <span class="hidden-word">of interest</span></button>
+            </div>
+    </form>
+
+    <div class="best-destinations-gallery gallery col-lg-10 col-sm-12">
+
+        <div class="best-destinations-box box">
+            <img class="best-destinations-photo photo" src="images/destinations_images/Maldives.svg" alt="Maldives">
+            <p class="home-hotel-name home-text">Maldives</p>
+
+        </div>
+        <div class="best-destinations-box box">
+            <img class="best-destinations-photo photo" src="images/destinations_images/Spain.svg" alt="Spain">
+            <p class="home-hotel-name home-text">Spain</p>
+
+        </div>
+        <div class="best-destinations-box box">
+            <img class="best-destinations-photo photo" src="images/destinations_images/Norway.svg" alt="Norway">
+            <p class="home-hotel-name home-text">Norway</p>
+
+        </div>
+        <div class="best-destinations-box box">
+            <img class="best-destinations-photo photo" src="images/destinations_images/Island.svg" alt="Island">
+            <p class="home-hotel-name home-text">Island</p>
+
+        </div>
+        <form>
+            <button class="best-destinations-book">Book now</button>
+        </form>
+
+        <div class="home-switcher">
+            <img class="home-switcher-dest" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
```

Comment:
![image](https://user-images.githubusercontent.com/36200303/113576287-f304c880-9627-11eb-9c5f-5a60efe21dc4.png)
icon should be centered